z_infra_provisioning/cloud_infra_center/ocp_upi/destroy-bastion.yaml

```diff
@@ -0,0 +1,71 @@
+# =================================================================
+# Copyright 2021 https://github.com/openshift/installer
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =================================================================
+
+# Required Python packages:
+#
+# ansible
+# openstackclient
+# openstacksdk
+
+- hosts: localhost
+  gather_facts: false
+
+  tasks:
+    - name: 'Import common yaml'
+      become: true
+      become_user: root
+      become_method: sudo
+      ansible.builtin.blockinfile:
+        dest: /etc/named.conf
+        state: absent
+        block: |
+          zone "{{ cluster_name }}.{{ base_domain }}" {
+                  type master;
+                  file "{{ cluster_name }}.{{ base_domain }}.zone";
+                  allow-query { any; };
+                  allow-transfer { none; };
+                  allow-update { none; };
+          };
+      notify:
+        - restart dns
+
+    - name: Add reverse name resolution to /etc/named.conf
+      become: true
+      become_user: root
+      become_method: sudo
+      ansible.builtin.blockinfile:
+        dest: /etc/named.conf
+        state: absent
+        block: |
+          zone "{{ bastion_reverse }}.in-addr.arpa" {
+                  type master;
+                  file "{{ bastion_reverse }}.in-addr.arpa.zone";
```

Comment:
it should be ` file "{{ bastion_reverse }}.in-addr.arpa.{{ cluster_domain_name }}.zone";`, right?